### PR TITLE
Exposed ability to blacklist intervals in CNV WDLs.

### PR DESCRIPTION
**Pull request by samuelklee**
This is a relatively simple change, but has not been tested.  The reviewer should make sure I haven't screwed it up!

### CODE REVIEW

File: scripts/cnv_wdl/cnv_common_tasks.wdl [hunk 1]
```diff
@@ -1,5 +1,6 @@
 task PreprocessIntervals {
```
Comment (LeeTL1220) [inline scripts/cnv_wdl/cnv_common_tasks.wdl]: Can you add a doc comment?

File: scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl [hunk 2]
```diff
@@ -28,6 +28,7 @@ workflow CNVSomaticPairWorkflow {
     ##################################
     File common_sites
     File intervals
+    File? blacklist_intervals
```
Comment (LeeTL1220) [inline scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl]: Can you add a doc comment?

File: scripts/cnv_wdl/somatic/cnv_somatic_panel_workflow.wdl [hunk 3]
```diff
@@ -23,6 +23,7 @@ workflow CNVSomaticPanelWorkflow {
     #### required basic arguments ####
     ##################################
     File intervals
```
Comment (LeeTL1220) [inline scripts/cnv_wdl/somatic/cnv_somatic_panel_workflow.wdl]: Ditto re: doc comment

**Comment by samuelklee** [comment]
Thanks, back to you @LeeTL1220!

**Pull request merged**